.gitmodules

```diff
@@ -16,3 +16,6 @@
 [submodule "depends/jsoncpp"]
 	path = depends/jsoncpp-sub
 	url = ../../open-source-parsers/jsoncpp.git
+[submodule "depends/curl"]
+	path = depends/curl
+	url = ../curl.git
```

Comment:
This should have `../../[GitHub username]/curl.git` so it doesn't try to load from whoever forks the DFHack repository's account.